Demo/CreateViewController.swift

```diff
@@ -9,15 +9,20 @@
 import UIKit
 
 class CreateViewController: UIViewController {
+    
+    // MARK: - Properties
+    
 	var demoModelController: DemoModelController?
 
+    // MARK: - Outlets
+    
 	@IBOutlet var titleTextField: UITextField!
 	@IBOutlet var subtitleTextField: UITextField!
 	@IBOutlet var widthTextField: UITextField!
 	@IBOutlet var heightTextField: UITextField!
 
-
-
+    // MARK: - Actions
+    
```

Comment:
Please eliminate trailing whitespace (the line is empty except for a whitespace character)